__tests__/app.test.js

```diff
@@ -148,4 +147,75 @@ describe('GET /api/articles/:article_id/comments task 6',() => {
             expect(res.body.comments).toEqual([])
         });
     });
+});
+
+//7
+describe('POST /api/articles/:article_id/comments task 6',() => {
+    test('responds with status code 201, and posts comment with appropriate data types',() => {
+        const reqarticle_id = 1;
+        const newComment = {username: 'butter_bridge', body: 'outstanding' }
+        return request(app).post(`/api/articles/${reqarticle_id}/comments`)
+        .send(newComment)
+        .expect(201)
+        .then(({body}) => {
+            expect(body.comment).toEqual(
+                expect.objectContaining({
+                    comment_id: 19,
+                    votes: 0,
+                    created_at: expect.any(String),
+                    author: 'butter_bridge',
+                    body: 'outstanding'
+                }));
+        });
+    });
+    test('responds with status code 206 if input is missing',() => {
+        const reqarticle_id = 1;
+        const newComment = {body: 'outstanding' }
+        return request(app).post(`/api/articles/${reqarticle_id}/comments`)
+        .send(newComment)
+        .expect(206)
```

Comment:
I'm not convinced by this choice of status code. 
everything in the 200 range indicates that the request has been successful, which this has not. 
whereas the 400 range indicates a problem or error on the user's side. I would suggest a `400: bad request` for missing content, as it makes the comment unpostable.